packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-radio-group/gux-form-field-radio-group.tsx

```diff
@@ -0,0 +1,139 @@
+import { Component, Element, h, JSX, State, Prop, Watch } from '@stencil/core';
+
+import { OnMutation } from '@utils/decorator/on-mutation';
+import { hasSlot } from '@utils/dom/has-slot';
+
+import {
+  GuxFormFieldError,
+  GuxFormFieldHelp,
+  GuxFormFieldLegendLabel,
+  GuxFormFieldFieldsetContainer
+} from '../../functional-components/functional-components';
+import { getSlotTextContent } from '@utils/dom/get-slot-text-content';
+
+import { trackComponent } from '@utils/tracking/usage';
+
+/**
+ * @slot group-label - Required slot for label tag
+ * @slot group-error - Optional slot for group-error message
```

Comment:
Thoughts on saying: `@slot group-error - Optional slot for error message` and `@slot group-help - Optional slot for help message`?